apps/LineRider/install

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+install_packages apt-transport-https dirmngr gnupg ffmpeg mono-complete || exit 1
```

Comment:
```suggestion
install_packages apt-transport-https dirmngr gnupg ffmpeg mono-devel || exit 1
```

`mono-devel` is already enough for LineRider to run. It is more lightweight than `mono-complete`.